src/api-service/__app__/onefuzzlib/agent_events.py

```diff
@@ -214,9 +214,6 @@ def on_worker_event_done(machine_id: UUID, event: WorkerDoneEvent) -> Result[Non
             node.debug_keep_node = True
             node.save()
     else:
-        logging.error(
```

Comment:
note, this is now handled by `task.mark_failed`.  Since we don't record errors on tasks already marked as failures, this moves the error longing to model our state handling.